tests/InputFiles/Cce/CharacteristicExtract.yaml

```diff
@@ -63,7 +63,7 @@ Cce:
   StartTime: 0.0
   EndTime: 1000.0
   BoundaryDataFilename: CceR0257.h5
-  H5IsBondiData: false
+  H5IsBondiData: False
```

Comment:
Why this change?

Reply:
Just to be consistent with the other text in the file (not sure why it wasn't that way in the first place)

Reply:
Ah ok. Change the ones at the top of the file in the `ResourceInfo` block as well just so the whole file is consistent.

Reply:
Done!